### DISCUSSION TITLE
Latest version of WalletKit

### DISCUSSION THREAD

**Discussion by vanhoanHoang** [walt-id/.github#69] [Help]
Dear team, 

I'm currently trying to setup Web Wallet by following the instructions at https://docs.walt.id/v/apps/solutions/web-wallet. However, the latest version of WalletKit in the docker-compose file does not give the same WalletKit as the version deployed by your team at https://portal.walt.id/

As a result, when I tried to get a VC offer (QR Code) from Issuer Portal on my machine and pasted the URI onto the Web Wallet, I got an error: credential_offer missed. This is comprehensible as the Issuer (of WalletKit version) on my machines does not give a good URI for the new WebWallet. 

Could you tell me how to fix this please ?

Thank you,
Hoan Hoang

**Comment by mikeplotean** [comment]
Hi, the docker-compose file was updated to reference the new tech stack. Could you please try it out and let us know if there are any issues with it?
